syncd/syncd_notifications.cpp

```diff
@@ -461,8 +461,70 @@ void processNotification(
 // that some notiffication arrived
 
 std::condition_variable cv;
-std::mutex queue_mutex;
-std::queue<swss::KeyOpFieldsValuesTuple> ntf_queue;
+
+class ntf_queue_t
+{
+public:
+    bool enqueue(swss::KeyOpFieldsValuesTuple msg);
+    bool tryDequeue(swss::KeyOpFieldsValuesTuple& msg);
+    size_t queueStats()
+    {
+        std::lock_guard<std::mutex> lock(queue_mutex);
+
+        return ntf_queue.size();
+    }
+
+private:
+    std::mutex queue_mutex;
+    std::queue<swss::KeyOpFieldsValuesTuple> ntf_queue;
+    const size_t limit = 300000;
```

Comment:
can you add some comments on why this number?

Reply:
I specified it in PR description. Please check. I'll add as comment as well.